packages/surf/package.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
Did you configure this package not to be publishable?

Reply:
Will adding `"private": true` to package.json be good enough ?

Reply:
Will adding "private": true to package.json be good enough ?

Reply:
Yes that should be good enough